app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivityCurrentPlace.kt

```diff
@@ -10,16 +10,18 @@ import android.widget.TextView
 import androidx.core.app.ActivityCompat
 import androidx.core.content.ContextCompat
 
-class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
+//Put it at the top of the file and converted to constants as that is their purpose. Also made public so that they can be accessed.
+public const val APP_ID = "c2b2b2a424915a775cacac7d33a2217a"
+public const val WEATHER_URL = "https://home.openweathermap.org/data/2.5/weather"
 
-    val APP_ID = "c2b2b2a424915a775cacac7d33a2217a"
-    val WEATHER_URL = "https://home.openweathermap.org/data/2.5/weather"
+public const val MIN_TIME: Long = 5000
+public const val MIN_DISTANCE = 1000f
+public const val REQUEST_CODE = 101
 
-    val MIN_TIME: Long = 5000
-    val MIN_DISTANCE = 1000f
-    val REQUEST_CODE = 101
+class MapsActivityCurrentPlace(var NameOfCity: TextView? = null) {
 
-    var location_provider = LocationManager.GPS_PROVIDER
+//Renamed without the underscore
+    var locationProvider = LocationManager.GPS_PROVIDER
```

Comment:
Just removed the _ as per naming conventions for the Kotlin Style Guide.

Reply:
Good.